test/build/config/undefined/undefined.test.js

```diff
@@ -0,0 +1,16 @@
+"use strict";
+const { resolve } = require("path");
+const { run } = require("../../../utils/test-utils");
+
+describe("config flag with undefined export config file", () => {
+  it("should throw error with no configuration or index file", async () => {
+    const { exitCode, stderr, stdout } = await run(__dirname, [
+      "-c",
```

Comment:
```suggestion
  it("should not throw error with no configuration or index file", async () => {
    const { exitCode, stderr, stdout } = await run(__dirname, [
      "--config",
```

Reply:
All tests seems to be using alias as well as not grammatically correct, will raise new PR to correct them in bulk.